lib/DDG/Spice/IndegoBikeshareLocations.pm

```diff
@@ -0,0 +1,34 @@
+package DDG::Spice::IndegoBikeshareLocations;
```

Comment:
Woah, didn't see you guys commented here! My page didn't update when I wrote that last comment. 

>I agree with @marianosimone City + Systemname seems best, e.g. PHLIndego for Philadelphia

Sounds like we're in agreement here then :+1: 

> Is package name length really an issue?

@marianosimone no really, it just makes for really lengthy callback and filenames